src/index.html

```diff
@@ -6,9 +6,55 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item" value="apple">
+            <a class="is-active nav__link" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item" value="samsung">
```

Comment:
```suggestion
      <a href="#home" class="logo">
        <img src="./images/logo.png" alt="logo">
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item" value="apple">
            <a class="is-active nav__link" href="#apple">
               Apple
             </a>
          </li>
          
          <li class="nav__item" value="samsung">
```
fix code style all over the page